vmm/src/device_manager.rs

```diff
@@ -1982,6 +1985,12 @@ impl DeviceManager {
             .sock
             .to_str()
             .ok_or(DeviceManagerError::CreateVsockConvertPath)?;
+
+        // Make sure the socket path does not exist before we try to create it.
+        if Path::new(socket_path).exists() {
```

Comment:
I'm sure it's unlinked on shutdown so i'm not sure why it would be there unless something has gone wrong?

Reply:
If the VM is simply killed with SIGKILL, it won't go through the Drop implementation.

Reply:
I removed it!